portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase

```diff
@@ -3731,6 +3731,689 @@ definition {
+		}
+
+		task ("Configure Assets to combine all segments in asset publisher and") {
+			AssetListsAdmin.enableCollectionsToCombineAssetsFromSegments(collectionType = "Manual");
+		}
+
+		task ("Navigate to collection created") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Manual Collection Test");
+		}
+
+		task ("Create a variation") {
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "First Segment");
+
+			AssetListsAdmin.selectAsset(
+				assetType = "Basic Web Content",
+				entryList = "WC_Title_1");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "First Segment");
+		}
+
+		task ("Create a second variation") {
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "Second Segment");
+
+			AssetListsAdmin.selectAsset(
+				assetType = "Basic Web Content",
+				entryList = "WC_Title_1");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "Second Segment");
+		}
+
+		task ("Create a public page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name",
+				type = "content");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+		}
+
+		task ("Add a Collection Display to page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Content Display",
+				fragmentName = "Collection Display");
+		}
+
+		task ("Select the manual collection in the Collection Display") {
+			PageEditor.editCollectionDisplay(
+				assetListName = "Manual Collection Test",
+				fragmentName = "Collection Display",
+				styleDisplay = "Bulleted List");
+
+			Button.clickPublish();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+		}
+
+		task ("Assert the asset appear only once in Collection Display Fragment") {
+			AssertTextEquals(
+				index = "1",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_1");
+
+			AssertElementNotPresent(
+				index = "2",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_1");
+		}
+
+		task ("Set collections configuration to default values") {
+			AssetListsAdmin.resetCollectionsToCombineAssetsFromSegments();
+		}
+	}
+
+	@description = "LPS-169312: Given a user belonging to Segment 1 and Segment 2
+	And the configuration to combine assets is enabled
+	When there's a variation of a dynamic collection targeting segment 1 and segment 2, and both are prioritized above the anyone variation
+	And there are some assets present in both variations**
+	Then the assets shown to the user will be ordered according to:
+	The asset priority (different from the variation priority)
+	The creationDate of the asset"
+	@priority = "3"
+	test ViewCombineContentOrderForMultisegmentDynamicCollection {
+		property portal.acceptance = "true";
```

Comment:
This test has priority 3 so it is not considered as acceptance. Remove this property.